docs/plugins.md

```diff
@@ -1720,7 +1729,7 @@ These plugins create heatmaps and heatmap-like visualizations from vector data.
 		</td><td>
 			A Leaflet plugin for rendering heatmaps and clusters from <a href='https://lucene.apache.org/solr/guide/6_6/spatial-search.html#SpatialSearch-HeatmapFaceting'>Solr's Heatmap Faceting</a>. High performance for millions of points or polygons.
 		</td><td>
-			<a href="https://github.com/mejackreed">Jack Reed</a> / 
```

Comment:
No worries for now, but watch out for accidental edits such as these, which, even if well-intentioned, makes a mess of the git history for files. Keep edits focused to what the commit is about!